CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## not released yet
 
+#### Added
+- Support for ACL flag. ([#26](https://github.com/peak/s5cmd/issues/26))
```

Comment:
Could you use the same section structure as the previous release please? I mean, Bugfixes, Features, Improvements etc.

Reply:
`Add [access control lists (ACLs)](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html) support to enable managing access to buckets and objects.`

Reply:
yeah, I should follow the pattern in changelog.md to keep consistency intact. thanks for pointing it out.